.circleci/config.yml

```diff
@@ -0,0 +1,65 @@
+version: 2
+jobs:
+  build-and-test:
+    working_directory: /ITKPhaseSymmetry-build
+    docker:
+      - image: insighttoolkit/module-ci:latest
+    steps:
+      - checkout:
+          path: /ITKPhaseSymmetry
+      - run:
+          name: Fetch CTest driver script
+          command: |
+            curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake -O
+      - run:
+          name: Configure CTest script
+          command: |
+            SHASNIP=$(echo $CIRCLE_SHA1 | cut -c1-7)
+            cat > dashboard.cmake << EOF
+            set(CTEST_SITE "CircleCI")
+            set(CTEST_BUILD_NAME "External-ITKPhaseSymmetry-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}-${SHASNIP}")
+            set(CTEST_BUILD_CONFIGURATION "MinSizeRel")
+            set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
+            set(CTEST_BUILD_FLAGS: "-j5")
+            set(CTEST_SOURCE_DIRECTORY /ITKPhaseSymmetry)
+            set(CTEST_BINARY_DIRECTORY /ITKPhaseSymmetry-build)
+            set(dashboard_model Experimental)
+            set(dashboard_no_clean 1)
+            set(dashboard_cache "
+            ITK_DIR:PATH=/ITK-build
+            BUILD_TESTING:BOOL=ON
+            ")
+            include(\${CTEST_SCRIPT_DIRECTORY}/itk_common.cmake)
+            EOF
+      - run:
+          name: Build and Test
+          no_output_timeout: 1.0h
+          command: |
+            ctest -j 2 -VV -S dashboard.cmake
+  package:
+    working_directory: ~/
+    machine: true
+    steps:
+      - checkout:
+          path: ~/ITKPhaseSymmetry
+      - run:
+          name: Fetch build script
+          command: |
+            curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
+            chmod u+x dockcross-manylinux-download-cache-and-build-module-wheels.sh
+      - run:
+          name: Build Python packages
+          no_output_timeout: 1.0h
+          command: |
+            cd ~/ITKPhaseSymmetry
+            ./dockcross-manylinux-download-cache-and-build-module-wheels.sh
+      - store_artifacts:
+          path: dist
+          destination: dist
+
+workflows:
+    version: 2
+    build-test-package:
+      jobs:
+        - build-and-test
+        - package
```

Comment:
No newline at and of file. Is that an issue?

Reply:
Thanks @dzenanz  ! Done in 6c736b7.